asynchbase/pom.xml

```diff
@@ -47,7 +47,7 @@ LICENSE file.
         </exclusion>
       </exclusions>
     </dependency>
-    
+
```

Comment:
I don't think we should include changes like formatting or whitespace removal into our 3rd party forks. They always make further upstream integration much more difficult.